### PR TITLE
feat: LUKSO, LUKSO testnet

### PR DESCRIPTION
**Pull request by zengzengzenghuy**
# What
Add LUKSO and LUKSO testnet to hyperlane-registry.

# Reference
https://docs.lukso.tech/networks/mainnet/parameters

### CODE REVIEW

**Comment by avious00** [comment]
@zengzengzenghuy thanks for the PR, reviewing now!

**Comment by bertux** [comment]
@zengzengzenghuy just run `yarn lint:fix` to fix the alphabetical sorting while waiting for my merged fix in the CLI to be released.
For prettier just run `yarn run prettier`

File: chains/lukso/logo.svg [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+</style>
+<path class="st0" d="M499.5,114.6L325,13.9C294.1-4,255.9-4,225,13.9L50.5,114.6c-30.9,17.9-50,50.9-50,86.6v201.5
+	c0,35.7,19.1,68.7,50,86.6L225,590.1c30.9,17.9,69.1,17.9,100,0l174.5-100.8c30.9-17.9,50-50.9,50-86.6V201.2
+	C549.5,165.5,530.5,132.5,499.5,114.6z M413.5,322l-51.9,89.9c-7.1,12.4-20.3,20-34.6,20H223.1c-14.3,0-27.5-7.6-34.6-20L136.5,322
+	c-7.1-12.4-7.1-27.6,0-40l51.9-89.9c7.1-12.4,20.3-20,34.6-20h103.8c14.3,0,27.5,7.6,34.6,20l51.9,89.9
+	C420.6,294.4,420.6,309.6,413.5,322z"/>
+<path class="st1" d="M413.5,322l-51.9,89.9c-7.1,12.4-20.3,20-34.6,20H223.1c-14.3,0-27.5-7.6-34.6-20L136.5,322
+	c-7.1-12.4-7.1-27.6,0-40l51.9-89.9c7.1-12.4,20.3-20,34.6-20h103.8c14.3,0,27.5,7.6,34.6,20l51.9,89.9
+	C420.6,294.4,420.6,309.6,413.5,322z"/>
+</svg>
```
Comment (paulbalaji) [inline chains/lukso/logo.svg:17]: ran through https://jakearchibald.github.io/svgomg/
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" xml:space="preserve" style="enable-background:new 0 0 550 604" viewBox="0 0 550 604"><path d="M499.5 114.6 325 13.9C294.1-4 255.9-4 225 13.9L50.5 114.6c-30.9 17.9-50 50.9-50 86.6v201.5c0 35.7 19.1 68.7 50 86.6L225 590.1c30.9 17.9 69.1 17.9 100 0l174.5-100.8c30.9-17.9 50-50.9 50-86.6V201.2c0-35.7-19-68.7-50-86.6zm-86 207.4-51.9 89.9c-7.1 12.4-20.3 20-34.6 20H223.1c-14.3 0-27.5-7.6-34.6-20l-52-89.9c-7.1-12.4-7.1-27.6 0-40l51.9-89.9c7.1-12.4 20.3-20 34.6-20h103.8c14.3 0 27.5 7.6 34.6 20l51.9 89.9c7.3 12.4 7.3 27.6.2 40z" style="fill:#fe005b"/><path d="m413.5 322-51.9 89.9c-7.1 12.4-20.3 20-34.6 20H223.1c-14.3 0-27.5-7.6-34.6-20l-52-89.9c-7.1-12.4-7.1-27.6 0-40l51.9-89.9c7.1-12.4 20.3-20 34.6-20h103.8c14.3 0 27.5 7.6 34.6 20l51.9 89.9c7.3 12.4 7.3 27.6.2 40z" style="fill:#fff1f8"/></svg>
```

File: chains/luksotestnet/logo.svg [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+</style>
+<path class="st0" d="M499.5,114.6L325,13.9C294.1-4,255.9-4,225,13.9L50.5,114.6c-30.9,17.9-50,50.9-50,86.6v201.5
+	c0,35.7,19.1,68.7,50,86.6L225,590.1c30.9,17.9,69.1,17.9,100,0l174.5-100.8c30.9-17.9,50-50.9,50-86.6V201.2
+	C549.5,165.5,530.5,132.5,499.5,114.6z M413.5,322l-51.9,89.9c-7.1,12.4-20.3,20-34.6,20H223.1c-14.3,0-27.5-7.6-34.6-20L136.5,322
+	c-7.1-12.4-7.1-27.6,0-40l51.9-89.9c7.1-12.4,20.3-20,34.6-20h103.8c14.3,0,27.5,7.6,34.6,20l51.9,89.9
+	C420.6,294.4,420.6,309.6,413.5,322z"/>
+<path class="st1" d="M413.5,322l-51.9,89.9c-7.1,12.4-20.3,20-34.6,20H223.1c-14.3,0-27.5-7.6-34.6-20L136.5,322
+	c-7.1-12.4-7.1-27.6,0-40l51.9-89.9c7.1-12.4,20.3-20,34.6-20h103.8c14.3,0,27.5,7.6,34.6,20l51.9,89.9
+	C420.6,294.4,420.6,309.6,413.5,322z"/>
+</svg>
```
Comment (paulbalaji) [inline chains/luksotestnet/logo.svg:17]: ran through https://jakearchibald.github.io/svgomg/
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" xml:space="preserve" style="enable-background:new 0 0 550 604" viewBox="0 0 550 604"><path d="M499.5 114.6 325 13.9C294.1-4 255.9-4 225 13.9L50.5 114.6c-30.9 17.9-50 50.9-50 86.6v201.5c0 35.7 19.1 68.7 50 86.6L225 590.1c30.9 17.9 69.1 17.9 100 0l174.5-100.8c30.9-17.9 50-50.9 50-86.6V201.2c0-35.7-19-68.7-50-86.6zm-86 207.4-51.9 89.9c-7.1 12.4-20.3 20-34.6 20H223.1c-14.3 0-27.5-7.6-34.6-20l-52-89.9c-7.1-12.4-7.1-27.6 0-40l51.9-89.9c7.1-12.4 20.3-20 34.6-20h103.8c14.3 0 27.5 7.6 34.6 20l51.9 89.9c7.3 12.4 7.3 27.6.2 40z" style="fill:#fe005b"/><path d="m413.5 322-51.9 89.9c-7.1 12.4-20.3 20-34.6 20H223.1c-14.3 0-27.5-7.6-34.6-20l-52-89.9c-7.1-12.4-7.1-27.6 0-40l51.9-89.9c7.1-12.4 20.3-20 34.6-20h103.8c14.3 0 27.5 7.6 34.6 20l51.9 89.9c7.3 12.4 7.3 27.6.2 40z" style="fill:#fff1f8"/></svg>
```

File: chains/lukso/metadata.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://explorer.execution.mainnet.lukso.network/api
+    family: blockscout
+    name: LUKSOExplorer
```
Comment (paulbalaji) [inline chains/lukso/metadata.yaml:5]: nit based on the name in the explorer home page
```suggestion
    name: LUKSO Mainnet Explorer
```

File: chains/luksotestnet/metadata.yaml [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://api.explorer.execution.testnet.lukso.network/api
+    family: blockscout
+    name: LUKSOExplorer
```
Comment (paulbalaji) [inline chains/luksotestnet/metadata.yaml:5]: nit based on the name in the explorer home page
```suggestion
    name: LUKSO Testnet Explorer
```

**Review by paulbalaji — COMMENTED** [review]
please run `yarn lint:fix` and `yarn prettier` :)

File: chains/lukso/logo.svg (hunk 1 above)
Comment (zengzengzenghuy) [reply chains/lukso/logo.svg]: Done

File: chains/luksotestnet/logo.svg (hunk 2 above)
Comment (zengzengzenghuy) [reply chains/luksotestnet/logo.svg]: Done

File: chains/lukso/metadata.yaml (hunk 3 above)
Comment (zengzengzenghuy) [reply chains/lukso/metadata.yaml]: Done

File: chains/luksotestnet/metadata.yaml (hunk 4 above)
Comment (zengzengzenghuy) [reply chains/luksotestnet/metadata.yaml]: Done

File: chains/lukso/metadata.yaml [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://explorer.execution.mainnet.lukso.network/api
+    family: blockscout
+    name: LUKSO Mainnet Explorer
+    url: https://explorer.execution.mainnet.lukso.network/
```
Comment (paulbalaji) [inline chains/lukso/metadata.yaml:6]:
```suggestion
    url: https://explorer.execution.mainnet.lukso.network
```

Comment (zengzengzenghuy) [reply chains/lukso/metadata.yaml]: Done

**Review by paulbalaji — DISMISSED** [review]

**Comment by paulbalaji** [comment]
please add a `minor` changeset with `yarn changeset add`

**Review by jmrossy — APPROVED** [review]
I've added a changeset for you. The additions lgtm!

**Pull request merged**